ROADMAP.md

```diff
@@ -8,73 +8,118 @@ not listed now does not mean it will not be considered for Antrea. We definitely
+* CLI tools to facilitate migration from another CNI: we will take care of
+  provisioning the correct network resources for your existing workloads.
+
+### Core networking features
+
+We are currently working on supporting VLAN tagging for Egress traffic. In the
+long term, we plan to add BGP support to the Antrea Agent, as it is a much
+requested feature.
+
+### Windows support improvements
+
+Antrea [supports Windows K8s Nodes](docs/windows.md). However, a few features
+including: Egress, NodePortLocal, IPsec encryption are not supported for Windows
+yet. We will continue to add more features for Windows (starting with Egress)
+and aim for feature parity with Linux. We encourage users to reach out if they
+would like us to prioritize a specific feature. While the installation procedure
+has improved significantly since we first added Windows support, we plan to keep
+on streamlining the procedure (more automation) and on improving the user
+documentation.
+
+### More robust FQDN support in Antrea NetworkPolicy
+
+Antrea provides a comprehensive network policy model, which builds upon
+Kubernetes Network Policies and provides many additional capabilities. One of
+them is the ability to define policy rules using domain names (FQDNs). We think
+there is some room to improve user experience with this feature, and we are
+working on making it more stable.
+
+### Implementation of new upstream NetworkPolicy APIs
+
+[SIG Network](https://github.com/kubernetes/community/tree/master/sig-network)
+is working on [new standard APIs](https://network-policy-api.sigs.k8s.io/) to
+extend the base K8s NetworkPolicy resource. We are closely monitoring the
+upstream work and implementing these APIs as their development matures.
+
+### Better network troubleshooting with packet capture
+
+Antrea comes with many tools for network diagnostics and observability. You may
+already be familiar with Traceflow, which lets you trace a single packet through
+the Antrea network. We plan on also providing users with the ability to capture
+live traffic and export it in PCAP format. Think tcpdump, but for K8s and
+through a dedicated Antrea API!
+
+### Multi-network support for Pods
+
+We recently added the SecondaryNetwork feature, which supports provisioning
+additional networks for Pods, using the same constructs made popular by
+[Multus](https://github.com/k8snetworkplumbingwg/multus-cni). However, at the
+moment, options for network "types" are limited. We plan on supporting new uses
+cases (e.g., secondary network overlays, network acceleration with DPDK), as
+well as on improving user experience for this feature (with some useful
+documentation).
+
+### L7 security policy
+
+Support for L7 NetworkPolicies was added in version 1.10, providing the ability
+to select traffic based on the application-layer context. However, the feature
+currently only supports HTTP traffic, and we plan to extend support to other
```

Comment:
L7 also supports TLS traffic since v1.13.
https://github.com/antrea-io/antrea/blob/main/docs/antrea-l7-network-policy.md

Reply:
thanks, I totally forgot about this one